tests/unit/dataactvalidator/test_b18_object_class_program_activity_1.py

```diff
@@ -0,0 +1,45 @@
+from factory import fuzzy
+from tests.unit.dataactcore.factories.staging import ObjectClassProgramActivityFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+
+_FILE = 'b18_object_class_program_activity_1'
+
+not_req_dict = {
+        "submission_id":fuzzy.FuzzyInteger(9999),
+        "object_class":fuzzy.FuzzyInteger(1000,9999),
```

Comment:
I needed different ranges than the defaults to test specifically 3-digit (0,999) or 4-digit (1000,9999) object classes.  I chose fuzzy over randint here just for consistency with the rest of the factory object.  I'll switch back to randint.